src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -269,6 +269,10 @@ watch(orderedSlides, (newSlides, oldSlides) => {
 	@apply tw-hidden;
 }
 
+.journey-card-carousel:deep(div:first-child) {
+	@apply tw-gap-2;
```

Comment:
This won't work for smaller screen size like iPhone SE.
My guess is better to follow smt like here: https://github.com/kiva/ui/blob/0082a12e6f75a0bec89db9671d3e6dca76de6348/src/components/MyKiva/BorrowerCarousel.vue#L231

Based on viewport size.